.editorconfig

```diff
@@ -7,6 +7,8 @@ indent_size = 4
 indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
+quote_type = single
```

Comment:
VSCode kept wanting to put double quotes which was against the current style of this repo so I added this until we have styleci setup